alexandria/storages/fields.py

```diff
@@ -56,8 +90,9 @@ def pre_save(self, instance, add):
                     "Set `ALEXANDRIA_FILE_STORAGE` to `alexandria.storages.s3.S3Storage`."
                 )
                 raise ImproperlyConfigured(msg)
-            storage = SsecGlobalS3Storage()
             if instance.encryption_status == File.EncryptionStatus.SSEC_GLOBAL_KEY:
-                self.storage = storage
+                self.storage = storages.create_storage(
+                    {"BACKEND": "alexandria.storages.backends.s3.SsecGlobalS3Storage"}
+                )
         _file = super().pre_save(instance, add)
         return _file
```

Comment:
the variable and the underscore thingy are both uneccessary

```suggestion
        return super().pre_save(instance, add)
```